notebooks/DrizzlePac/sky_matching/sky_matching.ipynb

```diff
@@ -4,14 +4,27 @@
    "cell_type": "markdown",
```

Comment:
<p>Please remove the unnecessary <code>else</code> statement. Since <code>continue</code> is the default behavior when the loop condition is met.</p><p>Let's use <code>f-string</code> for cleaner and more readable string formatting.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>